docs/java/activities.md

```diff
@@ -4,24 +4,65 @@ title: Activities in Java
 sidebar_label: Activities
 ---
 
-Activities can be invoked during Workflow execution. Similar to Workflows, Activities in Temporal
-Java SDK programming model are classes which implement an Activity Interface:
+## What is an Activity?
```

Comment:
Would it be ok to rewrite this paragraph some? If so let me know I can provide some suggestions

Reply:
Of course.

Reply:
updated. thanks!